doc/rst/technotes/reduceIntents.rst

```diff
@@ -165,20 +122,11 @@ or coforall loop. Here is an example of such a class:
 Future Work
 -----------
 
-* Implement reduce intents for cobegin statements.
-
-* Provide the other predefined reduction operators as reduce intents:
+* Switch to a light-weight interface for user-defined reductions.
+  The current proposal is discussed as a
```

Comment:
How about making this "discussed in a `GitHub issue`"? (Where the highlighted text is the link text).